devices_dex.go

```diff
@@ -0,0 +1,174 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type Data map[string]interface{}
```

Comment:
if this is being exported, i'd recommend namespacing it to be `DeviceDexData` or similar.

Reply:
That's a good point, thanks!